cityofaustin/atd-moped
# adds full name to navigation search

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/16832

## Testing
**URL to test:** 


https://deploy-preview-1334--atd-moped-main.netlify.app/

**Steps to test:**
- find or create a project that has a secondary name with a string that is not contained in the primary name
- search for the secondary name string in the nav search bar
- your result should appear, displaying the full name!

---
#### Ship list
